packages/dashboard/src/instance/instanceDetails.tsx

```diff
@@ -359,6 +374,13 @@ function getTreeOfNavigationItems(instance: GetInstanceQuery['instance']) {
 
   // if instance has video add it as the first item of the navigation tree
   if (instance?.results?.videoUrl) {
+
+    if (S3_PRESIGN_DOWNLOAD_URLS === true) {
+      const key = instance.results.videoUrl.split('/').pop()  // split url and get last item to get object key
+      const signedDownloadUrl = s3.getDownloadUrl(key)
+      instance.results.videoUrl = signedDownloadUrl
```

Comment:
This `split` doesn't work - it says `Property 'split' does not exist on type 'string'.`. How can I make this work?